include/aws/common/external/cJSON.h

```diff
@@ -28,10 +28,10 @@ THE SOFTWARE.
  *   (3) Remove cJSON_GetErrorPtr and global_error as they are not thread-safe.
  */
 
-/* clang-format off */
+/* NOLINTBEGIN */
 
```

Comment:
why are we reenabling clang format? if we disabling tidy, we should disable format as well

Reply:
there is a .clang-format config that disabled the format already.